code/modules/surgery/brain_surgery.dm

```diff
@@ -34,7 +34,7 @@
 		"[user] completes the surgery on [target]'s brain.")
 	if(target.mind && target.mind.has_antag_datum(/datum/antagonist/brainwashed))
 		target.mind.remove_antag_datum(/datum/antagonist/brainwashed)
-	target.adjustBrainLoss(-60)
+	target.adjustOrganLoss(ORGAN_SLOT_BRAIN, target.getOrganLoss(ORGAN_SLOT_BRAIN) - 60)	//we set damage in this case in order to clear the "failing" flag
```

Comment:
Are you sure it should be the organloss - 60 like this and not just -60?

Reply:
Thanks for catching that, it should be setOrganLoss!